spec/acceptance/plugin_spec.rb

```diff
@@ -1,27 +1,36 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 
 describe 'jenkins class', order: :defined do
-  PDIR = '/var/lib/jenkins/plugins'.freeze
+  # rubocop:todo RSpec/LeakyConstantDeclaration
+  PDIR = '/var/lib/jenkins/plugins' # rubocop:todo Lint/ConstantDefinitionInBlock, RSpec/LeakyConstantDeclaration
+  # rubocop:enable RSpec/LeakyConstantDeclaration
 
   # files/directories to test plugin purging removal of unmanaged files
-  FILES = [
+  # rubocop:todo RSpec/LeakyConstantDeclaration
```

Comment:
Can't you leave it off longer? So just `:todo` once and `:enable` once rather than 3 times?

Reply:
sure. that was autogenerated by rubocop